src/fe/fe_base.C

```diff
@@ -1521,6 +1521,17 @@ FEGenericBase<OutputType>::compute_proj_constraints (DofConstraints & constraint
   if (!elem->active())
     return;
 
+#ifdef LIBMESH_ENABLE_INFINITE_ELEMENTS
+    // for infinite elements, only the base is constrained.
```

Comment:
Is this comment an atavism?  We discussed that in 3D there would be constraints on the off-base DoFs too, right?

But then the code below seems over-zealous, since in *2D* it's true that only the finite element neighboring an infinite element needs to be constrained; the infinite element layer is conforming.

Reply:
oh yes, I forgot that one..